test/e2e/test_worker_config.py

```diff
@@ -21,6 +22,64 @@
 @pytest.mark.parametrize("operating_system", [os.environ["OPERATING_SYSTEM"]], indirect=True)
 class TestWorkerConfiguration:
 
+    def test_worker_requires_no_instance_profile(
+        self,
+        deadline_resources,
+        deadline_client: DeadlineClient,
+        worker_config: DeadlineWorkerConfiguration,
+        function_worker_factory: Callable[[DeadlineWorkerConfiguration], EC2InstanceWorker],
+    ) -> None:
+
+        # Create a EC2 worker with disallow-instance-profiles option for the worker agent
+        # Note that the EC2 instance is created with an instance profile, so no job will ever be picked up by this worker
+        function_worker_factory(
+            dataclasses.replace(
+                worker_config,
+                disallow_instance_profile="True",
```

Comment:
Is it expected to cause lint error?

```
test/e2e/test_worker_config.py:33: error: Unexpected keyword argument
"disallow_instance_profile" for "replace" of "DeadlineWorkerConfiguration"  [call-arg]
                dataclasses.replace(
                ^
Found 1 error in 1 file (checked 145 source files)
```

Reply:
I see it's due to an dependent change

https://github.com/aws-deadline/deadline-cloud-test-fixtures/blob/mainline/src/deadline_test_fixtures/deadline/worker.py#L96